code/modules/mining/equipment/kinetic_crusher.dm

```diff
@@ -444,3 +446,14 @@
 		chaser.monster_damage_boost = FALSE // Weaker cuz no cooldown
 		chaser.damage = 20
 		log_combat(user, target, "fired a chaser at", src)
+
+//Syndicate Trophy
+/obj/item/crusher_trophy/syndicate_trophy
+	name= "Syndicate trophy"
+	desc = "An odd super conducting material that the syndicate has stolen from the mitts of the ash lizards during a lavaland exploretory team deployment, it seems to cackle with power"
+	icon = 'icons/obj/mining.dmi'
```

Comment:
why isn't this a unique sprite

Reply:
i have the artistic abilities of an 8 year old

Reply:
grey scale the regular one then bro. syndicate is grey black and red!

Reply:
*crackle, not cackle